support-frontend/typescript-errors.json

```diff
@@ -3710,6 +3676,12 @@
             "message": "Object is possibly 'null'."
         }
     ],
+    "TS7016": [
+        {
+            "path": "assets/helpers/rendering/render.ts",
+            "message": "Could not find a declaration file for module 'preact/debug'. 'support-frontend/node_modules/preact/debug/dist/debug.js' implicitly has an 'any' type."
+        }
+    ],
```

Comment:
I'm not sure if this is something we added? We made sure to run `yarn` before generating the error file

Reply:
Is this the same error that was removed on lines 3135 & 3136 from the 'before' diff?

Reply:
Yeah it looks that way, I'm not sure why these errors were previously removed because it looks like we don't have the corresponding types module for `preact/debug`